discord/colour.py

```diff
@@ -329,7 +329,15 @@ def nitro_pink(cls: Type[CT]) -> CT:
         .. versionadded:: 2.0
         """
         return cls(0xF47FFF)
-
+    
+    @classmethod
```

Comment:
Unrelated to this PR, should be reverted and put in a different PR if you want it to be added.